src/seedu/addressbook/ui/TextUi.java

```diff
@@ -75,7 +57,7 @@ private boolean isCommentLine(String rawInputLine) {
      * @return command (full line) entered by the user
      */
     public String getUserCommand() {
-        out.print(LINE_PREFIX + "Enter command: ");
+        out.print(Formatter.LINE_PREFIX + "Enter command: ");
```

Comment:
errr.... here the UI is still doing the string processing/formatting (particularly code inside the `print()`)